code/data_preprocessing/genotype/VCF_QC.ipynb

```diff
@@ -70,13 +70,18 @@
    "cell_type": "markdown",
    "id": "another-carry",
    "metadata": {
-    "kernel": "SoS"
+    "kernel": "SoS",
+    "tags": []
    },
    "source": [
     "## Input\n",
     "\n",
     "1. The target VCF file to be QC\n",
+    "\n",
+    "    1.1 If the chromosome name of vcf to be qc is not chr*, please run dbsnp_annotate:1 of this module to fix it. Otherwise no need to run it for it is time consuming.\n",
```

Comment:
I think we should call these separate steps: `rename_chrs` and `dbsnp_annotate`. Please revise and implement them as separate steps